assets/pages/new-contributions-landing/components/ContributionSubmit.jsx

```diff
@@ -26,7 +26,7 @@ type PropTypes = {
 // ----- Render ----- //
 
 function ContributionSubmit(props: PropTypes) {
-  const currency: IsoCurrency = detect(props.countryGroupId);
+  const currency: IsoCurrency = fromCountryGroupId(props.countryGroupId) || 'GBP';
```

Comment:
Out of interest, why are we no longer first looking for the currency in the query parameters?

Reply:
I wasn't aware, perhaps sth to pick-up when I'm done with the logic.